src/types.ts

```diff
@@ -9,6 +18,12 @@ export interface Ref<T> {
 
 // GraphQL helper types
 export type SelectionSet = ReadonlyArray<SelectionNode>;
+export type GraphQLFlatType =
```

Comment:
This looks to be "any type which is not a list or non-null"

```
Exclude<GraphQLType, GraphQLWrappingType>;
```

Reply:
Oh awesome! Didn't know about `GraphQLWrappingType`. Either way that's super elegant